ggml/src/ggml-rpc/ggml-rpc.cpp

```diff
@@ -612,11 +612,17 @@ static void ggml_backend_rpc_buffer_clear(ggml_backend_buffer_t buffer, uint8_t
     GGML_ASSERT(status);
 }
 
+static void ggml_backend_rpc_buffer_memset_tensor(ggml_backend_buffer_t buffer, ggml_tensor * tensor, uint8_t value, size_t offset, size_t size) {
+    ggml_backend_rpc_buffer_context * ctx = (ggml_backend_rpc_buffer_context *)buffer->context;
+    std::vector<uint8_t> data(size, value);
+    ggml_backend_rpc_buffer_set_tensor(buffer, tensor, data.data(), offset, size);
```

Comment:
this implementation is inefficient as it potentially sends large amount of data over the network; we should add a dedicated RPC command for this

Reply:
You are right. I can implement that.